cli/installer/kubernetes.go

```diff
@@ -0,0 +1,211 @@
+		dnf:      "sudo dnf -y install helm",
+		homebrew: "brew install helm",
+		macIntelChipManual: `
+			curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+			chmod 700 get_helm.sh
+			./get_helm.sh
+		`,
+		macAppleChipManual: `
+			curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+			chmod 700 get_helm.sh
+			./get_helm.sh
+		`,
+		windows: "",
+		other:   "",
+	}).exec(ui)
+}
+
+func installKubectl(ui UI) {
+	(cmd{
+		sudo:          true,
+		notConfirmMsg: "No worries, you can try installing kubectl manually. See https://kubernetes.io/docs/tasks/tools/#kubectl",
+		installDocs:   "https://kubernetes.io/docs/tasks/tools/#kubectl",
+		args:          map[string]string{},
+		apt: `
+			sudo apt-get update
+			sudo apt-get install -y ca-certificates curl apt-transport-https
+
+			sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+			echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
+
+			sudo apt-get update
+			sudo apt-get install -y kubectl
+		`,
+		yum: `
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+sudo yum -y install kubectl
+		`,
+		dnf: `
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+[kubernetes]
+name=Kubernetes
+baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
+enabled=1
+gpgcheck=1
+gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+EOF
+sudo dnf -y install kubectl
+		`,
+		homebrew: "brew install helm",
+		macIntelChipManual: `
+			curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/amd64/kubectl"
+			chmod +x ./kubectl
+			sudo mv ./kubectl /usr/local/bin/kubectl
+			sudo chown root: /usr/local/bin/kubectl
+		`,
+		macAppleChipManual: `
+			curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/arm64/kubectl"
+			chmod +x ./kubectl
+			sudo mv ./kubectl /usr/local/bin/kubectl
+			sudo chown root: /usr/local/bin/kubectl
+		`,
+		windows: "",
+		other:   "",
+	}).exec(ui)
+}
+
+func installMinikube(ui UI) {
+	(cmd{
+		sudo:          true,
+		notConfirmMsg: "No worries, you can try installing minikube manually. See https://minikube.sigs.k8s.io/docs/start/",
+		installDocs:   "https://minikube.sigs.k8s.io/docs/start/",
+		args:          map[string]string{},
+		apt: `
+			sudo apt update
+			sudo apt install curl -y
+			curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
```

Comment:
We cannot assume architecture. we need to detect it and use accordingly

Reply:
Done